etc/apache-mod_proxy_fcgi.conf.template

```diff
@@ -0,0 +1,77 @@
+###
```

Comment:
Note we also have a `robot.template` so this plugin would likely be affected by a `templates` subdirectory.